docs/reference/commandline/update.md

```diff
@@ -13,7 +13,7 @@ keywords: "resources, update, dynamically"
      will be rejected.
 -->
 
-## update
+# update
```

Comment:
This heading should be removed as it's now generated from the metadata (see my previous comment)